test/domain/relation/_fixtures/relation.fixture.ts

```diff
@@ -47,5 +48,17 @@ await database.createRecord(RECORD_TYPE_RELATION, relation5);
 await database.createRecord(RECORD_TYPE_RELATION, relation6);
 await database.createRecord(RECORD_TYPE_RELATION, relation7);
 
-export { CREATOR0, CREATOR1, CREATOR2, CREATOR3, CREATOR4, CREATOR5, establish, explore, getFollowers, getFollowing, RELATION1, RELATION2, RelationAlreadyExists, RelationData, REQUESTER1, REQUESTER2, SortOptions };
+const QUERY0: RecordQuery = {
```

Comment:
The name 'QUERY0' is very abstract and doesn't describe the query. Can you give a more meaningful name? This also goes for the query below of course.